src/helpers/MiscFunctions.cpp

```diff
@@ -456,6 +456,25 @@ SWorkspaceIDName getWorkspaceIDNameFromString(const std::string& in) {
     return result;
 }
 
+bool shouldWraparound(const WORKSPACEID id1, const WORKSPACEID id2) {
```

Comment:
function is only used once, I don't think it needs to be global in miscfunctions. can be moved as a static fn in monitor.cpp